src/client/components/ConsentsSubHeader.tsx

```diff
@@ -40,224 +48,177 @@ const h1ResponsiveText = css`
+      text-align: left;
+
+      /**
+       * This allows us to force only "Create password" to wrap on mobile
+       */
+      ${until.mobileLandscape} {
+        white-space: normal;
+        width: 60px;
+      }
     }
-    &::after {
-      background-color: ${neutral[60]};
+
+    &:last-of-type > span {
+      right: -${CIRCLE_RADIUS}px;
+      left: auto;
+      text-align: right;
     }
+
     &::before {
-      border: 2px solid ${neutral[60]};
-      background-color: white;
-    }
-    &:first-child::before {
-      left: 0;
-    }
-    &:last-child::before {
-      background-color: ${neutral[60]};
+      content: '';
+      border-radius: 50%;
+      width: ${CIRCLE_DIAMETER}px;
+      height: ${CIRCLE_DIAMETER}px;
+      position: absolute;
+      left: -${CIRCLE_RADIUS}px;
+      /* TODO: apply this to psuedo elements globally */
+      box-sizing: border-box;
+      ${status === 'active' &&
+      `
+        background: ${neutral[100]};
+        border: ${COMPLETED_BORDER_SIZE}px solid ${COMPLETED_COLOR};
+      `}
+      ${status === 'pending' &&
+      `
+        background: ${neutral[100]};
+        border: ${PENDING_BORDER_SIZE}px solid ${PENDING_COLOR};
+      `}
+      ${status === 'complete' &&
+      `
+        background: ${brand[400]}
+      `}
     }
-    &:last-child::after {
-      border: 2px solid ${neutral[60]};
-      background-color: white;
-    }
-  }
-`;
 
-const ul = css`
-  display: flex;
-  list-style: none;
-  justify-content: space-between;
-  height: 54px;
-  padding: 0;
-  margin: 0;
+    ${status === 'active' && textSans.xxsmall({ fontWeight: 'bold' })}
+  `;
+};
+
+const progressWrapper = css`
+  padding: 0 ${CIRCLE_RADIUS}px;
+  overflow: hidden;
 `;
 
-const maxContentOverride = (isLastPage: boolean) => css`
-  width: ${isLastPage ? 'max-content' : 'initial'};
+const hiddenText = css`
+  display: none;
 `;
 
-const PageProgression = ({
-  pages,
-  current,
-  cssOverrides,
-}: {
-  pages: string[];
-  current?: string;
-  cssOverrides?: SerializedStyles | SerializedStyles[];
-}) => {
-  const active = current ? pages.indexOf(current) : 0;
-  const getClassName = (i: number) => {
-    switch (true) {
-      case i === active:
-        return 'active';
-      case i < active:
-        return 'complete';
-      default:
-        return '';
-    }
-  };
+export const ConsentsSubHeader = ({ autoRow, title, current }: Props) => {
+  const active = current
+    ? (CONSENTS_PAGES_ARR as string[]).indexOf(current)
+    : 0;
+
+  const pageProgression = current && (
+    <div css={[progressWrapper, autoRow()]}>
+      <ol css={ol(active)}>
+        {CONSENTS_PAGES_ARR.map((page, index) => {
+          const status: PageStatus = (() => {
+            if (index === active) {
+              return 'active';
+            }
+            if (index < active) {
+              return 'complete';
+            }
+            return 'pending';
+          })();
+          const screenReaderText = {
```

Comment:
is there a case for adding `pending` to this list?

Reply:
I was working from this example: https://www.w3.org/WAI/tutorials/forms/multi-page/

I think it should be implied from the order of the list items and the status of the preceding items?

Reply:
That makes sense, that's a really helpful doc, thanks for linking